CHANGELOG.md

```diff
@@ -66,7 +66,7 @@ Changelog
 * add new `contributionTypes` property to `properties` parameter ([#136])
 * add new `/contributions/latest/count` and `/contributions/latest/count/density` endpoints ([#174])
 * add new `contributionType` parameter for contributions aggregation ([#174])
-* contribution extraction endpoints now return the `@contributionChangesetId` as a separate field ([#200])
+* contributions extraction endpoints now return the `@contributionChangesetId` as a separate field ([#200])
```

Comment:
I am not sure if using two or more plurals in a compound noun is right. You say bus stops and not busses stops, even if more busses stop at the same bus stop (there are for sure some exceptions). Since I am not a mother tongue I would ask them (not sure they can give us the right answer because this is an even more specific case of three nouns, or better it would be to ask philologists, they know the rule, if any) or let the compound noun how it was.

Reply:
The endpoint doesn't extract a single contribution but is meant to return multiple contribution**s**. That's why I think this is correct.